Sources/Plugins/OAPlugin.mm

```diff
@@ -362,7 +362,7 @@ + (NSString *)onGetMapObjectsLocale:(NSObject *)object preferredLocale:(NSString
 
 + (void)registerCustomPoiFilters:(NSMutableArray<OAPOIUIFilter *> *)poiUIFilters
 {
-    for (OAPlugin *p in [self getEnabledPlugins])
```

Comment:
[self getEnabledPlugins] crush app on start. 

Replaced id to [self.class getAvailablePlugins] like here:
https://github.com/osmandapp/OsmAnd/blob/9086e72d8914e7d9da3767b382413aee67dc395e/OsmAnd/src/net/osmand/plus/OsmandPlugin.java#L909